store.go

```diff
@@ -75,12 +91,15 @@ func (s *DefaultConfigStore) Alias(from, to string) {
 // one didn't previously exist.
 func (s *DefaultConfigStore) SetLevel(level ConfigLevel, key string, value interface{}) {
 	s.setIfNotExists(level, key, value)
+	s.writeIfLog(key, value)
```

Comment:
We should probably include some context here about whether we're logging a read or a write, no?